### ISSUE TITLE
Uses Passive Event Listeners to Improve Scrolling Performance

### ISSUE THREAD

**Issue by rtarnaud** [Semantic-Org/Semantic-UI#6233]
According to the Google devs, it is a good practice to flag every wheel, mousewheel, touchstart, and touchmove event listener with the passive attribute, see [here](https://developers.google.com/web/tools/lighthouse/audits/passive-event-listeners).
There is a simple feature detection test to implement in order to see if the browser supports this flag [here](https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md#feature-detection) and it seems that the only place in Semantic UI where this flag needs to be passed is in one of the helper functions for managing events.

**Comment by jlukic** [comment]
I've been thinking about this previously. It's most likely contingent on jQuery supporting passive listeners https://github.com/jquery/jquery/issues/2871

**Comment by rtarnaud** [comment]
Yes, sorry you are right. The helper I was mentioning is indeed part of Jquery.
